features/zones/Create.feature

```diff
@@ -97,14 +112,34 @@ Feature: zones / create
       """
       body:
         zone:
-          enabled: true
+          enabled: !!js/undefined
         zoneInsight:
           subscriptions:
             - connectTime: '2020-07-28T16:18:09.743141Z'
               disconnectTime: !!js/undefined
       """
     Then the "$connected" element exists
 
+  Scenario: The zone is deleted whilst waiting to be connected and shows an error
+    Given the environment
+      """
+      KUMA_SUBSCRIPTION_COUNT: 0
+      """
```

Comment:
**Question**: Why does this need 0 subscriptions?

Reply:
Because the scenario is 'whilst waiting to be connected'. It "should" fire the successful request after `And I click` but before `And the URL: 404`, but thats not clear enough in the test and possibly not guaranteed to happen before the `And the URL`.

I've added a couple more assertions in https://github.com/kumahq/kuma-gui/pull/1754/commits/fb09a7070bda66bd8e1bb23c6377c7ac7d15741d so we can guarantee that at least one successful (but disconnected) request happens.